SingleResponsibilityPrinciple/Controllers/Invoice.cs

```diff
@@ -0,0 +1,8 @@
+public class Invoice
```

Comment:
Nice @johne5s ! We traditionally split the models out. Their one job is to represent an object, nothing else.